supervisor/api/os.py

```diff
@@ -212,3 +222,41 @@ async def boards_other_info(self, request: web.Request) -> dict[str, Any]:
             )
 
         return {}
+
+    @api_process
+    async def config_swap_info(self, request: web.Request) -> dict[str, Any]:
+        """Get swap settings."""
+        if not self.coresys.os.available or self.coresys.os.version < "15.0":
+            raise HTTPNotFound()
```

Comment:
Looking at the status codes, this is the only one that makes sense. That being said, we can provide a better explanation then this so I think we should. Something like telling them this API requires the system be HAOS with version >= 15.0 . And perhaps log this as well.